backend/iam/sso/saml/views.py

```diff
@@ -135,14 +135,14 @@ def dispatch(self, request, organization_slug):
             email = auth._nameid
             user = User.objects.get(email=email)
             idp_first_name = auth._attributes.get(
-                "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname"
+                "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname", [""]
```

Comment:
As `idp_last_name` is a string, why fall back to `[""]` and not simply `""`?

Reply:
because `_attributes` returns an array containing the `givenname` or the `lastname`.
`idp_last_name` is a string because we get index 0 of this array directly.

Reply:
Kinda didn't read the following line, my bad. LGTM